src/completers/env.zsh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env zsh
+
+_fzf_complete_env() {
+    local arguments=("${(Q)${(z)@}[@]}")
+    arguments=($(_fzf_complete_trim_env "${${(q)arguments[2,-1][@]}}"))
+    local cmd=${(Q)arguments[1]}
+
+    if (( $+functions[_fzf_complete_$cmd] )); then
+        LBUFFER=${LBUFFER/env /}
+        _fzf_complete_$cmd ${@/env /}
```

Comment:
Is this expansion safe again and again?

Reply:
Here is safe.

Reply:
I don't think so. Here is a function that simply prints the arguments delimited by a comma:
```zsh
print-args() {
    echo ${(pj:, :)@}
}

print-args '' 2 '' 4 # => ', 2, , 4'
```
When this function is called from a function like this, it looks like it drops empty strings:
```zsh
call-print-args() {
    print-args ${@/env /}
}

call-print-args '' 2 'env ' 4 # => '2, 4'
```